added introduction.md

zapirkon opened the pull request.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

Commit:
added introduction.md

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,4 @@
+INTRODUCTION
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Commit:
introduction.md update

Commit:
added kills

zapirkon committed:
Merge pull request #1 from zapirkon/deadly_skills

added kills

zapirkon committed:
Merge pull request #2 from zapirkon/master

Merge pull request #1 from zapirkon/deadly_skills

Commit:
Added deadly skills

Commit:
Merge remote-tracking branch 'refs/remotes/origin/deadly_skills' into deadly_skills

Commit:
another updatE

zapirkon committed:
Merge pull request #3 from zapirkon/deadly_skills

Deadly skills

Commit:
addddd agaiiinnn

Commit:
added skill as requested

codeschool-kiddo committed:
Added new rule about coffee

Commit:
spelling corrected

zapirkon committed:
Merge pull request #5 from zapirkon/new_rules

Too many people are spilling coffee

Commit:
Merge branch 'deadly_skills'

Commit:
bad code instead of kills

Commit:
Merge branch 'release_branch_1.0'

Commit:
added grievance to close #6

grievance modified and fixes #6

grievance modified third time and fixes #6

grievance modified third time and again fixes #6

please fixed #6

pleeeee fixed #6

Commit:
added grievance to close #6

grievance modified and fixes #6

grievance modified third time and fixes #6

grievance modified third time and again fixes #6

please fixed #6

pleeeee fixed #6

zapirkon committed:
Merge branch 'kill_list' of https://github.com/zapirkon/dojo_rules into kill_list

zapirkon committed:
this fixes #6

zapirkon committed:
Merge pull request #7 from zapirkon/kill_list

Kill list

zapirkon committed:
Update README.md

zapirkon committed:
updated info

the info is updated

zapirkon committed:
Update README.md

zapirkon committed:
Update README.md

zapirkon committed:
Update README.md
